<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Piotr Esden-Tempski</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qgovernor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="OpenBLDCTabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Open-BLDC</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="GovernorBakendTab">
       <attribute name="title">
        <string>Governor Backend</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QGroupBox" name="registerGroupBox">
          <property name="title">
           <string>Register Contents</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTableView" name="registerTableView">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="palette">
              <palette>
               <active>
                <colorrole role="Base">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>222</red>
                   <green>230</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="AlternateBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>91</red>
                   <green>132</green>
                   <blue>254</blue>
                  </color>
                 </brush>
                </colorrole>
               </active>
               <inactive>
                <colorrole role="Base">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>222</red>
                   <green>230</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="AlternateBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>91</red>
                   <green>132</green>
                   <blue>254</blue>
                  </color>
                 </brush>
                </colorrole>
               </inactive>
               <disabled>
                <colorrole role="Base">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>212</red>
                   <green>208</green>
                   <blue>200</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="AlternateBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>91</red>
                   <green>132</green>
                   <blue>254</blue>
                  </color>
                 </brush>
                </colorrole>
               </disabled>
              </palette>
             </property>
             <property name="font">
              <font>
               <family>Monaco</family>
               <pointsize>11</pointsize>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderStretchLastSection">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderStretchLastSection">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="graphicGroupBox">
          <property name="title">
           <string>Graphic Display</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="outputGroupBox">
          <property name="title">
           <string>Output</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QTableView" name="outputTableView">
             <property name="palette">
              <palette>
               <active>
                <colorrole role="Base">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>222</red>
                   <green>230</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="AlternateBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>91</red>
                   <green>132</green>
                   <blue>254</blue>
                  </color>
                 </brush>
                </colorrole>
               </active>
               <inactive>
                <colorrole role="Base">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>222</red>
                   <green>230</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="AlternateBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>91</red>
                   <green>132</green>
                   <blue>254</blue>
                  </color>
                 </brush>
                </colorrole>
               </inactive>
               <disabled>
                <colorrole role="Base">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>212</red>
                   <green>208</green>
                   <blue>200</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="AlternateBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>91</red>
                   <green>132</green>
                   <blue>254</blue>
                  </color>
                 </brush>
                </colorrole>
               </disabled>
              </palette>
             </property>
             <property name="font">
              <font>
               <family>Monaco</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="alternatingRowColors">
              <bool>false</bool>
             </property>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QGroupBox" name="inputGroupBox">
          <property name="title">
           <string>Input</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QTableView" name="inputTableView">
             <property name="palette">
              <palette>
               <active>
                <colorrole role="Base">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>222</red>
                   <green>230</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="AlternateBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>91</red>
                   <green>132</green>
                   <blue>254</blue>
                  </color>
                 </brush>
                </colorrole>
               </active>
               <inactive>
                <colorrole role="Base">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>222</red>
                   <green>230</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="AlternateBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>91</red>
                   <green>132</green>
                   <blue>254</blue>
                  </color>
                 </brush>
                </colorrole>
               </inactive>
               <disabled>
                <colorrole role="Base">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>212</red>
                   <green>208</green>
                   <blue>200</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="AlternateBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>91</red>
                   <green>132</green>
                   <blue>254</blue>
                  </color>
                 </brush>
                </colorrole>
               </disabled>
              </palette>
             </property>
             <property name="font">
              <font>
               <family>Monaco</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="alternatingRowColors">
              <bool>false</bool>
             </property>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>914</width>
     <height>22</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>true</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Device</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Q&amp;uit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>&amp;Preferences...</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qgovernor.qrc">
     <normaloff>:/icons/icons/connect-32x32.png</normaloff>
     <normalon>:/icons/icons/disconnect-32x32.png</normalon>:/icons/icons/connect-32x32.png</iconset>
   </property>
   <property name="text">
    <string>Connect...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qgovernor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
